api/swagger.yaml

```diff
@@ -6174,7 +6174,7 @@ paths:
         - name: "buildargs"
           in: "query"
           description: "JSON map of string pairs for build-time variables. Users pass these values at build-time. Docker uses the buildargs as the environment context for commands run via the `Dockerfile` RUN instruction, or for variable expansion in other `Dockerfile` instructions. This is not meant for passing secret values. [Read more about the buildargs instruction.](https://docs.docker.com/engine/reference/builder/#arg)"
-          type: "integer"
+          type: "string"
```

Comment:
As a follow-up we could add an example value for this;

```json
{"ARG_ONE":"foo","ARG_TWO":"bar"}
```

or (URL encoded, but I don't think we need to show the encoded value):

```
%7B%22ARG_ONE%22%3A%22foo%22%2C%22ARG_TWO%22%3A%22bar%22%7D
```

Reply:
Thanks. I'd be happy to add that here. I was a bit confused by the language so a better description would be 💯

Reply:
Yes, that'd be great!

Looks like (unfortunately), ReDoc doesn't support example values for query parameters (at least, they don't seem to show in the output; tried both `example:` and `x-example`), so the example value best is included in the description.

For readability, it's probably best to change the description text to use the "multi line" format (also a bit easier to work with);

```yaml
description: |
  JSON map of string pairs for build-time variables. Users pass these values at
  build-time. Docker uses the buildargs as the environment context for commands
  run via the `Dockerfile` RUN instruction, or for variable expansion in other
  `Dockerfile` instructions. This is not meant for passing secret values.
  [Read more about the buildargs instruction](https://docs.docker.com/engine/reference/builder/#arg)"
```